sdks/python/setup.py

```diff
@@ -107,13 +107,17 @@ def get_version():
     'crcmod>=1.7,<2.0',
     # Dill doesn't guarantee comatibility between releases within minor version.
     'dill>=0.3.0,<0.3.1',
+    'facets-overview>=1.0.0,<2',
```

Comment:
This change is significantly expanding the required Beam dependencies. These dependencies are by default not used by existing users and also would not be needed in execution environments.

I suggest making another extras_require similar to gcp and group all interactive dependency requirements under that group.

Reply:
Thanks for the suggestions!
Yes, I feel the same way but didn't know what to do about it. Will make the change.